ukis_pysat/data.py

```diff
@@ -77,6 +74,23 @@ def __init__(self, datahub, datadir=None, datadir_substr=None):
     def __enter__(self):
         return self
 
+    def init_catalog(self, catalog=None):

```

Comment:
we should handle the case where we just have a collection of loose STAC items stored as JSON files in a directory (e.g., when we process an image and store a metadata item along with it). in such case from a user's perspective I would expect that if I set "catalog" to the root directory where the items are stored that I get a STAC catalog (that I can query) with all valid items in this directory. Perhaps this should be split into two separate calls: 1. init_catalog() initilaize an empty catalog in this case; 2. fill_catalog_from_directory() or similar to fill the catalog with items.

Reply:
self.catalog should be self.api (see comment above).